validation/validateUserForm.ts

```diff
@@ -0,0 +1,42 @@
+import { IValidation } from "../models/validationModel";
+
+export const validateForm = <IValidation>(value: string, id: string) => {
+  if (id === "firstName") {
+    if (value.length <= 2) {
+      return {
+        message: "First name longer than two characters is required",
+        id: id,
+        isActive: true,
+        isDirty: true,
+      };
+    } else {
```

Comment:
else is unnecessary. The return in the if above will automatically stop the rest of the function from running.

Reply:
done